ext/vscode/docs/commands.md

```diff
@@ -0,0 +1,70 @@
+# VS Code Extension Commands
+
+This document describes the commands provided by the Azure Developer CLI Visual Studio Code extension.
+
+## Command List
+
+Most commands are available in the Command Palette (Ctrl+Shift+P or Cmd+Shift+P) and are prefixed with "Azure Developer CLI (azd)".
```

Comment:
Similar to the feedback in the other file, this kinda implies that it's a list of all the commands, but in fact is just a list of the one command.